test/functional/platform/Document.spec.js

```diff
@@ -142,11 +143,92 @@ describe('Platform', () => {
 
     it('should fetch created documents array', async () => {
```

Comment:
Is it outdated?

Reply:
Why?

Reply:
It says we fetch document array, but we are fetching just a single doc.